WasatchNET/EEPROM.cs

```diff
@@ -289,7 +290,22 @@ public float[] wavecalCoeffs
             set
             {
                 EventHandler handler = EEPROMChanged;
-                _wavecalCoeffs = value;
+                if (wavecalCoeffs == null || value.Length == wavecalCoeffs.Length)
+                    _wavecalCoeffs = value;
+                else
+                {
+                    int index = 0;
+                    while (index < value.Length)
+                    {
+                        wavecalCoeffs[index] = value[index];

```

Comment:
It seems weird that you're using wavecalCoeffs[index] here instead of _wavecalCoeffs[index].  I guess I see what's happening, but...it's weird.

Reply:
Yeah, silly mistake